source/access/mfa_login.rst

```diff
@@ -3,23 +3,221 @@
 Multi Factor Authentication (MFA)
 =================================
 
-|KUL| Multi Factor Authentication (MFA) is an augmented level of security which, as
-the name suggests, requires multiple steps when authenticating. The following
-two factors are necessary to connect to the KU Leuven clusters:
+|KUL| Multi Factor Authentication (MFA) is an augmented level of security.
+As the name suggests, MFA requires additional steps with human intervention
+when authenticating.
+MFA is mandatory for accessing KU Leuven infrastructures.
+This document provides two methods to connect to the KU Leuven clusters with MFA.
 
-- A valid private key
-- Access to a VSC-associated university/institution account
+The first method, *without* an :ref:`SSH agent <SSH agent>`, is easier to get started with,
+but requires some repetitive steps with each connection, hence inconvenient on a long run.
 
-This approach ensures that if your private key is compromised, the person who
-has unauthorized access to it will still not be able to abuse your VSC account.
-This document provides two methods to connect to the KU Leuven clusters with
-MFA. The first method, *without* an agent, is easier to get started with, but requires
-some repetitive steps with each connection. The second method, *with* an agent,
-requires some more effort to set up initially but will be easier when you need
-multiple connections or you are using some specific GUI applications to connect. 
+The second method involves an :ref:`SSH agent <SSH agent>` and requires a few more
+steps to set up. This approach is however more convenient in the long run because
+a single authentication will suffice for multiple connections throughout the day,
+without further intervention.
 
-.. toctree::
+Below, we elaborate on these two approaches.
 
-  mfa_quickstart
-  mfa_guide
+.. note::
+
```

Comment:
This note is valid for all sites, independent from our MFA setup. So it would make sence to add this one level higher, on the general authentication page.

Reply:
Done in d71f23b